android-lib/src/androidTest/java/com/kinvey/androidTest/store/data/ClientBuilderTest.java

```diff
@@ -174,14 +175,13 @@ public void testSetTimeoutRequest() throws IOException {
     public void testSetInstanceId() throws IOException {
         client = new Client.Builder(mContext).setInstanceID("TestInstanceId").build();
         assertEquals(Constants.PROTOCOL_HTTPS + "TestInstanceId" + Constants.HYPHEN + Constants.HOSTNAME_API + "/", client.getBaseUrl());
-        assertEquals(Constants.PROTOCOL_HTTPS + "TestInstanceId" + Constants.HYPHEN + Constants.HOSTNAME_AUTH + "/", client.getMICHostName());
+        assertEquals(Constants.PROTOCOL_HTTPS + "TestInstanceId" + Constants.HYPHEN + Constants.HOSTNAME_AUTH + "/", client.getMicHostName());
```

Comment:
try use `@JvmName("MICHostName")` to keep compatibility with the previous version

Reply:
Used `@JvmName` converted to old method name for `Java` classes.